.aspect/workflows/config.yaml

```diff
@@ -5,53 +5,138 @@ workspaces:
         icon: js
         label: rules_js
         tasks:
-            format:
```

Comment:
Did you intend to remove buildifier and gazelle?

Reply:
From this list of `tasks` yes. New bug in rosetta when there are workspaces meant I had to bump these two tasks to the root `tasks` list at the bottom.